pypeit/core/wavecal/echelle.py

```diff
@@ -83,6 +84,9 @@ def predict_ech_wave_soln(angle_fits_params, ech_angle_coeffs, ech_angle, order_
     for iord, order in enumerate(order_vec):
         # Index of the order in the total order vector used cataloguing the fits in the coeff arxiv
         indx = order - angle_fits_params['order_min']
+        # check if the order is in the arxiv, if not skip this order
+        if indx < 0 or indx >= angle_fits_params['norders']:
+            continue
```

Comment:
Issue a warning, or is this done elsewhere?

Reply:
I was debating a lot about adding a warning here (I added it and removed it many times). But the order vector that is used here is padded, meaning there are more orders in this vector than there are in the observations. So I thought that a warning may create more confusion than helping.